web/server/vue-cli/src/assets/userguide/userguide.md

```diff
@@ -14,429 +14,759 @@ on GitHub</a>
+This page lists the analysis result for the given run.
+
+![Reports](images/reports/reports.png)
+
+Every report has the following fields:
+- `Report ID`
+- `Unique report identifier` (repot hash). You can find more information how
+these hashes are generated [here](https://github.com/Ericsson/codechecker/blob/master/docs/analyzer/report_identification.md).
+- `File path`: file path where the report is found.
+- `Line and column numbers` of the report where the bug is.
+- `Checker name` which found the report during the analysis.
+- `Checker message`: description about the report.
+- `Analyzer name` which found the problem during the analysis.
+- [`Severity`](#severity-levels)
+- `Bug path length`: length of the bug path. Reports with longer bug path
+are more difficult to understand.
+- [`Latest review status`](#review-status)
+- [`Latest detection status`](#detection-status)
+- `Detection date`: the storage date when the report was stored to the server
+for the first time.
+- `Fix date`: storage date when the report is dissapeared from a run update.
+
+## Review status
+Reports can be assigned a review status of the following values:
+- <span class="customIcon review-status-unreviewed"></span> **Unreviewed**
+(_default_): Nobody has seen this report.
+- <span class="customIcon review-status-confirmed"></span> **Confirmed**:
+This is really a bug.
+- <span class="customIcon review-status-false-positive"></span>
+**False positive**: This is not a bug. Before marking a bug false positive
+you should read the [false positive how to](https://github.com/Ericsson/codechecker/blob/master/docs/analyzer/false_positives.md).
+- <span class="customIcon review-status-intentional"></span> **Intentional**:
+This report is a bug but we don't want to fix it.
+
+![Review statuses](images/reports/review_statuses.png)
+
+Review statuses are connected to
+[report hashes](https://github.com/Ericsson/codechecker/blob/master/docs/analyzer/report_identification.md).
+If the same report can be found in multiple runs it will have the same review
+status.
+
+It can be changed on the [GUI](#change-review-status) or by using
+[source code comments ](https://github.com/Ericsson/codechecker/blob/master/docs/analyzer/user_guide.md#source-code-comments)
+(*codechecker_false_positive*, *codechecker_confirmed*, etc.)
+
+**Note**: source code comment is stronger and can overwrite the value in the
+database
 
-## <a name="userguide-compare-runs"></a> Compare runs
-Calculates difference between two analyses of the project, showing which bugs
-have been fixed and which are newly introduced.
+## Detection status
+The detection status is the state of a bug report in a run.
+storing the results of a run from scratch then each report has	
+- The detection status of a report is connected to the **report ID**. It means
+that every report has a separate detection status.
+- It will be *calculated automatically* by the server on run storage events.
```

Comment:
Please add a note that the detection status is calculated based on the reports in the last storage and the current storage.